recipe/conda_build_config.yaml

```diff
@@ -51,18 +48,13 @@ m2w64_cxx_compiler:            # [win]
 m2w64_fortran_compiler:        # [win]
   - m2w64-toolchain            # [win]
 
-cuda_compiler:
-  - None
-  - nvcc                       # [(linux or win64) and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
-  - cuda-nvcc                  # [(linux or win64) and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
 cuda_compiler_version:
   - None
-  - 11.8                       # [(linux or win64) and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
-  - 12.4                       # [linux and ppc64le and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
-  - 12.6                       # [((linux and not ppc64le) or win64) and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
+  - 12                         # [(linux or win64) and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
```

Comment:
I don't want to switch to free-floating minor for 12.x in an unrelated PR. If you think that's a good idea, please open an issue and we can discuss it there.

Reply:
Right we also discussed this previously. The reason we pin this here is to avoid picking up incomplete CUDA Toolkit upgrades

We do plan to bump this version whenever the next CUDA Toolkit is fully available on conda-forge